services/mobileappbuilder/index.md

```diff
@@ -9,11 +9,15 @@ copyright:
 {:shortdesc: .shortdesc}
 {:screen:.screen}
 {:codeblock:.codeblock}
+{:note:.deprecated}
+
+**This service is deprecated:** All instances of this service are deprecated. Existing instances can be used until 11 August 2016. For more information, see the [deprecation announcement blog](https://developer.ibm.com/bluemix/2016/07/12/mobile-app-builder-now-part-bluemix-mobile-dashboard/){: new_window}.
+{:deprecated}
```

Comment:
From experience, I think you need a space between {: and deprecated}

Like this:
{: deprecated}